pilot/cmd/pilot-agent/status/server.go

```diff
@@ -294,7 +294,18 @@ func (s *Server) handlePprofTrace(w http.ResponseWriter, r *http.Request) {
 	pprof.Trace(w, r)
 }
 
+const (
+	ReadinessTimeout = time.Second * 3 // Default Readiness probe timeout. It is set the same in helm charts.
```

Comment:
I am not sure a static makes sense here since it can be changed?

Reply:
This is currently a const in the yaml. Without a way to mutate it.

Reply:
Users can mutate anything they want, we allow arbitrary overrides. And users can always make their own Deployments

Reply:
https://github.com/istio/istio/blob/374fcdcfa040a27c4ddf6adfc621f559f1361d00/pilot/cmd/pilot-agent/status/util/stats.go#L39 

This is copied here

Reply:
https://github.com/istio/istio/blob/374fcdcfa040a27c4ddf6adfc621f559f1361d00/manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml#L529